aries-site/src/pages/components/menu.mdx

```diff
@@ -42,6 +47,164 @@ Accessibility features such as screen reader messages are automatically built in
+    <Box height="xsmall">
+      <MenuGroupingExample bestPractice={false} />
+    </Box>
+  </Example>
+</BestPracticeGroup>
+
+### Ordering
+
+Frequency of use should guide the order in which menu items are presented.
+
+When menu items are grouped, order items within that group by frequency.
+Additionally, place the most frequently used group at the top of the menu.
+This anticipates what the user is looking for and allows them to locate the
+item as their eye scans the items top to bottom.
+
+Exception: [Dangerous actions](#dangerous-actions) should be placed as the
+last item in the menu.
+
+### Dangerous actions
+
+Dangerous or destructive actions altering a state which cannot
+be recovered, such as delete, should be placed as the last item in the menu
+with a visual separator for the dangerous actions grouping so that they are
+distinct from all other actions.
+
+Selecting a dangerous action should lead to double confirmation.
+
+<BestPracticeGroup>
+  <Example
+    bestPractice={{
+      type: 'do',
+      message:
+        'Always separate a dangerous action into its own group and place it at the end. Clicking on a dangerous action should lead to a double confirmation.',
+    }}
+  >
+    {/* Box allows menu to be presented with drop open, overriding Example align/justify center */}
+    <Box height="small">
+      <MenuDangerousExample />
+    </Box>
+  </Example>
+  <Example
+    bestPractice={{
+      type: 'dont',
+      message: `Don't leave dangerous actions grouped with other actions.`,
+    }}
+  >
+   {/* Box allows menu to be presented with drop open, overriding Example align/justify center */}
+    <Box height="small">
+      <MenuDangerousExample bestPractice={false} />
+    </Box>
+  </Example>
+</BestPracticeGroup>
+
+### Common groupings in HPE applications
+
+Many HPE applications provide the ability for customers to configure, modify,
+and act upon resources in their environment.
+
+For applications in this context, groups of menu items can be thought of as
+[configuring](#configuring), [executing](#executing), or [transferring](#transferring).
+
+#### Configuring
+
+Actions allowing a user to set and modify attributes of a resource.
+
+- View [resource] details
+- Edit [resource]
+- Add [item] to [resource]
+- Remove [item] from [resource]
+- Assign [resource] to [another resource]
+- Add [resource] to group
+
+#### Executing
+
+Actions providing the ability for a user to act upon a resource.
+
+- Launch [resource]
+- Power on/off [resource]
+- Reset [resource]
+- Update/upgrade software/firmware version for [resource]
+
+#### Transferring
+
+Actions enabling a user to share or consume data related to a resource.
+
+- Download [resource] data
+- Share [resource] with [user]
+- Send [resource] to [user]
+- Notify [user] about [resource]
+
+TODO: add a series of examples showcasing tangible resources (e.g. servers, clusters, etc.)
+
+#### Batch actions menu related to a selection of servers
```

Comment:
I liked the idea of adding a small description here about which about the above groupings this menu is representing / why one group is on top of the other

Reply:
just pushed changes which include descriptions